planetary-social/nos
# Implement NIP-96 and NIP-98 for file uploads

## Issues covered
#1208

## Description
Uploads photos and videos using the file storage API from nostr.build as described in [NIP-96](https://github.com/nostr-protocol/nips/blob/master/96.md).

## How to test
1. Tap Post
2. Tap the image upload button
3. Select an image
4. Observe URL
5. Post the post
6. Observe your image in a post

## Screenshots/Video

https://github.com/planetary-social/nos/assets/59564/c9d221cd-cba7-42d5-8faa-e207d481d7d0
